frontend/src/Admin/queries/portfolioPeriods.graphql

```diff
@@ -0,0 +1,17 @@
+query PortfolioPeriods {
```

Comment:
@BrodrickS This query grabs all the existing PortfolioPeriods. (Only works with the Admin by this PR)